src/jit/lower.cpp

```diff
@@ -2875,8 +2875,9 @@ void Lowering::InsertPInvokeMethodProlog()
     firstBlockRange.InsertBefore(insertionPoint, LIR::SeqTree(comp, storeSP));
     DISPTREERANGE(firstBlockRange, storeSP);
 
-#endif // !_TARGET_X86_
+#endif // !defined(_TARGET_X86_) && !defined(_TARGET_ARM_)
 
+#if !defined(_TARGET_ARM_)
     // --------------------------------------------------------
```

Comment:
Ok, I missed that "return initRegs". Yes, looks like the setting of calleeSavedFP is not necessary for ARM.

---
In reply to: [103448440](https://github.com/dotnet/coreclr/pull/9825#discussion_r103448440) [](ancestors = 103448440)